backends/cuda-reg/ceed-cuda-reg-basis.c

```diff
@@ -123,7 +127,7 @@ inline __device__ void interp1d(const CeedInt nelem, const int transpose,
     for(int comp = 0; comp < BASIS_NCOMP; comp++) {
       if (!transpose) {
         const int sizeU = P1D;
-        readDofs(bid, tid, comp, sizeU, nelem, d_U, r_V);
+        readQuads(bid, tid, comp, 0, sizeU, nelem, d_U, r_V);
```

Comment:
It's the same layout for e and q now, so it would the same function with different names, as far as I could tell.

Reply:
From what I can see they don't take the same number of arguments, I think it would be cleaner to have:
```c++
inline void readDofs(const int bid, const int tid, const int comp, const int sizeU, const int nelem, const double *U, double *V)
{
   readQuads(bid, tid, comp, 0, sizeU, nelem, U, V);
}
```
and the same for `writeDofs`.

Reply:
agreed. done